### PR TITLE
[CI:ALL] Bump to v5.2.0-rc3

### PR DESCRIPTION
**Pull request by mheon**
As the title says

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by mheon** [comment]
Eeek it's against the wrong branch

**Comment by mheon** [comment]
Alright, fixed, it's against 5.2 now

**Comment by mheon** [comment]
@containers/podman-maintainers PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## 5.2.0
 ### Features
+- Podman now supports `libkrun` as a backend for creating virtual machines on MacOS. The `libkrun` backend has the advantage of allowing GPUs to be mounted into the virtual machine to accelerate tasks. The default backend remains `applehv`.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:5]: It's a nit as almost everyone writes it "MacOS", but officially, it's "macOS"

**Comment by TomSweeneyRedHat** [comment]
LGTM 
assuming happy tests

**Comment by mheon** [comment]
Restarted one machine flake

**Comment by baude** [comment]
assuming a flake ... be sure it isnt

/lgtm

**Comment by edsantiago** [comment]
> assuming a flake ... be sure it isnt

You've been missing out. It's ALWAYS a flake. podman-machine is failing ~ 2 out of 3 runs. Sometimes more.

**Pull request merged**